SUMMARY.md

```diff
@@ -70,3 +70,11 @@
 * [Bug Bounty](miscellaneous/bug-bounty-program.md)
 * [Past paid bounties](miscellaneous/past-paid-bounties.md)
 * [Contact us](miscellaneous/contact-us.md)
+
+## Supported Networks
```

Comment:
Should these be a part of the Safe Smart Accounts section instead? Also, can we add an overview document with some context?

Reply:
Yes, they should

Reply:
We should also mention in the overview that:
- Currently our default version is still `v1.3.0` as `v1.4.1` is just deployed in a few networks and not supported in the Safe Transaction Service and the [Safe{Wallet}](app.safe.global) interface